tenzoMath/tenzoMath.cpp

```diff
@@ -1,149 +1,521 @@
 #include "tenzoMath.h"
+#include <fstream>
+#include <thread>
+#include <chrono>
+#include "poly34.h"
+#include <cmath>
+#include <string>
+#include <vector>
+#include <tchar.h>
+#include <ctime>
 
-TenzoMath::TenzoMath()
+void printCvMat(cv::Mat t, const char str[] = "some matrix")
 {
-	_positions = {
-		{
-			{0, 0, 0, 0, -90, 0},{0, 0, 0, 0, 0, 0},{0, 0, 0, 0, 0, 180},{0, 0, 0, 0, 90, 180},
-			{0, 0, 0, 0, 0, 270},{0, 0, 0, 0, 0, 90}
-		}
-	};
+    std::cout << str << ":\n";
+    for (int i = 0; i < t.rows; ++i)
+    {
+        for (int j = 0; j < t.cols; ++j)
+            if (abs(t.at<double>(i, j)) < 0.001)
+                std::cout << 0 << ' ';
+            else
+                std::cout << t.at<double>(i, j) << ' ';
+        std::cout << std::endl;
```

Comment:
(**must be changed, repeatedly**): Don't use `std::endl` if you don't need to flush buffer! Use `'\n'` instead.